src/features/info/search/actions.ts

```diff
@@ -178,9 +178,7 @@ const fetchSearch = createAsyncThunk(
               : item.name?.match(searchExpr) || item.provider.match(searchExpr)
           )
           .map((item: AppMarketplaceApp) => appToSearchItem(item)),
-        partners.content.map((item: { businessPartner: BusinessPartner }) =>
-          businessPartnerToSearchItem(item.businessPartner)
-        ),
+        partners.content.map((item: any) => businessPartnerToSearchItem(item)),
```

Comment:
define and use types or interfaces

Reply:
As discussed, this is needed for unit test case.